apps/encryption/lib/Crypto/Crypt.php

```diff
@@ -248,26 +251,40 @@ private function encrypt($plainContent, $iv, $passPhrase = '', $cipher = self::D
 	}
 
 	/**
-	 * return Cipher either from config.php or the default cipher defined in
+	 * return cipher either from config.php or the default cipher defined in
 	 * this class
 	 *
 	 * @return string
 	 */
-	public function getCipher() {
-		$cipher = $this->config->getSystemValue('cipher', self::DEFAULT_CIPHER);
-		if (!isset(self::SUPPORTED_CIPHERS_AND_KEY_SIZE[$cipher])) {
-			$this->logger->warning(
-				sprintf(
-					'Unsupported cipher (%s) defined in config.php supported. Falling back to %s',
-					$cipher,
-					self::DEFAULT_CIPHER
-				),
-				['app' => 'encryption']
-			);
-			$cipher = self::DEFAULT_CIPHER;
+	private function getCachedCipher() {
+		if (!isset($this->currentCipher)) {
```

Comment:
```suggestion
		if ($this->currentCipher === null) {
```

nitpick: if you flip the expression to !== then you can do an early return and save the indentation in the rest of the method ;)